website/content/en/docs/olm-integration/quickstart-bundle.md

```diff
@@ -111,10 +112,35 @@ Now we're ready to test and deploy the Operator with OLM.
 
 ### Testing bundles
 
-<!-- TODO(jmrodri): `run bundle` usage here -->
+In order to test our bundle in a cluster with [OLM installed](#enabling-olm).
```

Comment:
Since this is a guide, the reader is expected to have installed OLM already after reading the `Enabling OLM` section.
```suggestion
```